lib/reminder.js

```diff
@@ -0,0 +1,56 @@
+var debug = require('debug')('reminder');
+var hinterval = require('human-interval');
+var conf = require('./conf');
+var db = require('./db');
+var random = require('./random');
+var unansweredDB = db('prompts-unanswered');
+
+// Convert string durations into ms.
+['timeBetweenReminders', 'reminderJobInterval']
+.forEach(function(key) {
+  var hr = conf.get(key);
+  var ms = hinterval(conf.get(key));
+  conf.set(key, ms);
+  debug('transforming', key, hr, ms);
+});
+
+var prompt = function() {
+  var prompts = conf.get('reminderPrompts');
+  return prompts[random().nextInt(0, prompts.length)];
+};
+
+var scan = function(reminderHook, done) {
+  unansweredDB.createReadStream()
+    .on('data', function(data) {
+      var nextReminder = new Date(data.value.lastReminder + conf.get('timeBetweenReminders'));
+      if (nextReminder < Date.now()) {
```

Comment:
This probably acts fine but is an example of something I don't always trust about JS, since it's coercing the nextReminder to a number. Instead I'd usually do:

```
var nextReminder = data.value.lastReminder + conf.get('timeBetweenReminders');
if (nextReminder < Date.now()) ...
```